app/screens/AppNavigator/screens/Dex/PoolSwap/PoolSwapScreen.test.tsx

```diff
@@ -37,50 +37,23 @@ jest.mock("../../../../../hooks/wallet/TokensAPI", () => ({
     }]
 }));
 
-describe('poolswap page', () => {
```

Comment:
moved tests to e2e, lack of all the context provider make it too tedious to fix this unit test